azure-pipelines.yaml

```diff
@@ -251,3 +255,5 @@ stages:
                     test_pattern: apim.test.ts
                   ${{ else }}:
                     test_pattern: pipeline.test.ts
+
+name: "$(finalAction) - ${{ parameters.location }} - SDS Azure Platform"
```

Comment:
is location needed?

Reply:
sure -can remove it, kept it as it is as only want to set the initial action